VERSIONS.md

```diff
@@ -1,99 +1,100 @@
-| 3.5.1         | 3.14.0      | 4.6.0           | 2.0.0          | Not compatible                |
-| 3.5.0         | 3.13.0      | 4.4.0           | 1.10.0         | Not compatible                |
-| 3.4.2         | 3.12.8      | 4.3.3           | 1.9.3          | Not compatible                |
-| 3.4.1         | 3.12.3      | 4.3.1           | 1.9.1          | Not compatible                |
-| 3.4.0         | 3.12.3      | 4.3.0           | 1.9.0          | Not compatible                |
-| 3.3.0         | 3.12.2      | 4.3.0           | 1.8.0          | Not compatible                |
-| 3.2.0         | 3.11.1      | 4.2.1           | 1.7.1          | Unity IAP 3.1.0 < 3.3.0 (BC3) |
-| 3.1.1         | 3.10.7      | 4.0.5           | 1.6.2          | Unity IAP 3.1.0 < 3.3.0 (BC3) |
-| 3.1.0         | 3.10.6      | 4.0.4           | 1.6.1          | Unity IAP 3.1.0 < 3.3.0 (BC3) |
-| 3.0.1         | 3.9.2       | 4.0.1           | 1.5.1          | Unity IAP 3.1.0 < 3.3.0 (BC3) |
-| 3.0.0         | 3.9.2       | 4.0.1           | 1.5.0          | Unity IAP 3.1.0 < 3.3.0 (BC3) |
-| 2.3.1         | 3.7.5       | 3.5.2           | 1.4.5          | Unity IAP 3.1.0 < 3.3.0 (BC3) |
-| 2.3.0         | 3.7.2       | 3.5.2           | 1.4.4          | Unity IAP 3.1.0 < 3.3.0 (BC3) |
-| 2.2.1         | 3.4.0       | 3.2.0           | 1.2.0          | Unity IAP < 3.1.0             |
-| 2.2.0         | 3.4.0       | 3.2.0           | 1.2.0          | Unity IAP < 3.1.0             |
-| 2.1.0         | 3.2.2       | 3.1.0           |                | Unity IAP < 3.1.0             |
-| 2.0.9         | 3.0.1       | 3.0.4           | 1.0.6          | Unity IAP < 3.1.0             |
-| 2.0.8         | 3.0.1       | 3.0.4           | 1.0.6          | Unity IAP < 3.1.0             |
-| 2.0.7         | 3.0.1       | 3.0.4           | 1.0.6          | Unity IAP < 3.1.0             |
-| 2.0.6         | 3.0.1       | 3.0.4           | 1.0.6          | Unity IAP < 3.1.0             |
-| 2.0.5         | 3.0.1       | 3.0.3           | 1.0.6          | Unity IAP < 3.1.0             |
-| 2.0.4         | 3.0.0       | 3.0.3           | 1.0.1          | Unity IAP < 3.1.0             |
-| 2.0.3         | 3.0.0       | 3.0.2           | 1.0.1          | Unity IAP < 3.1.0             |
-| 2.0.0         | 3.0.0       | 3.0.2           | 1.0.1          | Unity IAP < 3.1.0             |
-| 1.2.2         | 2.5.0       | 2.4.1           |                |                               |
-| 1.2.1         | 2.5.0       | 2.3.0           |                |                               |
+| Version       | iOS version | Android version | Common version | Play Billing Library version | Unity IAP compatibility       |
+| ------------- | ----------- | --------------- | -------------- | ---------------------------- | ----------------------------- |
+| 6.10.0        | 4.43.2      | 7.12.0          | 11.1.0         | 6.2.1                        | Unity IAP 4.12.0+ (BC6)       |
```

Comment:
I thought about also adding links to releases for all these releases (as added in the plugin)... But I think it's ok to just add them moving forward.